File: Python/Longest Common Prefix/LongestCommonPrefix.py
class Solution:
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        if(len(strs)==0):
            return ""
        str0 = strs[0]
        for i in range(len(str0)):
            s0 = str0[i]
            for j in range(1, len(strs)):
                if(i == len(strs[j]) or strs[j][i] != s0):
                    return str0[:i]
        return str0

File: C#/Longest Common Prefix/LongestCommonPrefix.cs
public class Solution {
    public string LongestCommonPrefix (string[] strs) {
        if (strs.Length == 0 || strs == null) return "";
        string str = strs[0];
        for (int i = 0; i < strs[0].Length; i++) {
            char s = str[i];
            for (int j = 1; j < strs.Length; j++) {
                if (i == strs[j].Length || strs[j][i] != s) {
                    return str.Substring (0, i);
                }
            }
        }
        return str;
    }
}